packages/prisma/migrations/20230518084145_add_feature_flag_google_workspace/migration.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO
+  "Feature" (slug, enabled, description, "type")
+VALUES
+  (
+    'google-workspace-directory',
+    false,
+    'Enable Google Workspace Directory integration - Syncing of users and groups from Google Workspace to users teams.',
+    'OPERATIONAL'
+  ) ON CONFLICT (slug) DO NOTHING;
```

Comment:
Disable this by default - we need to wait for google to re-verify our app for this specific scope - Otherwise whenever a user hits connect calendar - They will just see permission denied.